src/viewmodel/prototype/get.js

```diff
@@ -62,6 +63,12 @@ export default function Viewmodel$get ( keypath, options ) {
 		value = wrapped.get();
 	}
 
+	if ( keypath.isRoot && options.fullRootGet ) {
+		for ( key in this.mappings ) {
+			value[ key ] = this.mappings[ key ].getValue();
+		}
```

Comment:
...having said that, maybe mutating the data object isn't harmful. If we're discouraging people from interacting directly with it anyway, then perhaps it's okay, since `ractive.get('mappedKeypath')` will always be intercepted by the mapping anyway, there doesn't seem to be any risk of breaking anything else. Hmm